src/shared/components/FileUpload/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Storage, Project, NexusFile } from '@bbp/nexus-sdk-legacy';
 import { StorageCommon } from '@bbp/nexus-sdk-legacy/lib/Storage/types';
 import { CreateFileOptions } from '@bbp/nexus-sdk-legacy/lib/File/types';
 import { labelOf } from '../../utils';
-import { UploadFile } from 'antd/lib/upload/interface';
+import { UploadFile, RcCustomRequestOptions } from 'antd/lib/upload/interface';
```

Comment:
What does this have to do with the yarn migration?

Reply:
The types changed, probably because of version differences

Reply:
gotcha!